daemon/containerio/cio.go

```diff
@@ -0,0 +1,202 @@
+package containerio
```

Comment:
why we just copy containerd's Fifo io code to pouch, can we reuse containerd's code ? :)

Reply:
@HusterWan Actually the code in containerd can't meet our need, we need to some modifications :)